app/src/main/java/com/pvp/app/ui/screen/task/GeneralTaskCreationForm.kt

```diff
@@ -0,0 +1,247 @@
+        Text(
+            text = "Create a General Task",
+            style = TextStyle(
+                fontSize = 24.sp,
+                fontWeight = FontWeight.Bold,
+                letterSpacing = 0.15.sp
+            ),
+            modifier = Modifier.padding(bottom = 16.dp)
+        )
+
+        TextField(
+            value = title,
+            onValueChange = { title = it },
+            label = { Text("Title") },
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(8.dp)
+        )
+
+        TextField(
+            value = description,
+            onValueChange = { description = it },
+            label = { Text("Description") },
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(8.dp)
+        )
+
+        Row(
+            verticalAlignment = Alignment.CenterVertically,
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(8.dp)
+        ) {
+            Text(
+                text = "Start Date: ${startDate.toLocalDate()}",
+                style = TextStyle(fontSize = 16.sp),
+                modifier = Modifier
+                    .weight(1f)
+                    .padding(end = 8.dp)
+            )
+
+            Button(
+                onClick = {
+                    isDatePickerDialogOpen.value = true
+                },
+                modifier = Modifier.wrapContentWidth()
+            ) {
+                Text("Pick Date")
+            }
+        }
+
+        if (isDatePickerDialogOpen.value) {
+            DatePickerDialog(
+                onDismissRequest = {
+                    isDatePickerDialogOpen.value = false
+                },
+                confirmButton = {
+                    TextButton(
+                        onClick = {
+                            isDatePickerDialogOpen.value = false
+                            val instant = datePickerState.selectedDateMillis?.let {
+                                Instant.ofEpochMilli(it)
+                            }
+                            startDate = LocalDateTime.ofInstant(instant, ZoneId.systemDefault())
+                        }
+                    ) {
+                        Text("OK")
+                    }
+                },
+                dismissButton = {
+                    TextButton(
+                        onClick = {
+                            isDatePickerDialogOpen.value = false
+                        }
+                    ) {
+                        Text("CANCEL")
+                    }
+                }
+            ) {
+                DatePicker(
+                    state = datePickerState
+                )
+            }
+        }
+
+        Button(
+            onClick = {
+                generalTasksManager.addTask(
+                    title = title,
+                    description = description,
+                    startDate = startDate
+                )
+                title = ""
```

Comment:
A single line is needed after [:168](https://github.com/PVP-K410/Calendar/pull/5/files#diff-5cb4157cc3199595b3f890972f66692335490de0d32dd64c40c87ffd8b102454R168) to separate different blocks of code to increase readability.